app/views/owners/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= form_for(@owner) do |f| %>
+  <%= f.label  :full_name %><br>
+  <%= f.text_field :first_name, placeholder: "First name..." %>
+  <!-- < f.label  :middle_name %> -->
+  <%= f.text_field :middle_name, placeholder: "Middle name..." %>
+  <!-- < f.label  :last_name %> -->
+  <%= f.text_field :last_name, placeholder: "Last name..." %><br>
+  <%= f.label  :address %><br>
+  <%= f.text_field :address, placeholder: "Street name/number..."%>
+  <!-- < f.label  :city %> -->
+  <%= f.text_field :city, placeholder: "City..."%>
+  <!-- < f.label  :state %> -->
+  <%= f.text_field :state, placeholder: "State..." %>
+  <!-- < f.label  :zip %> -->
+  <%= f.text_field :zip, placeholder: "Zipcode..."%><br>
+  <%= f.label  :years_lived_at_home %>
+  <%= f.text_field :year, class: "form-control" %>
+  <%= f.label  :email %>
+  <%= f.text_field :email, class: "form-control" %>
+  <%= f.label  :phone %>
+  <%= f.text_field :phone, class: "form-control" %>
+  <%= f.label  :Emergency_contact %><br>
+  <%= f.text_field :em_contact_name, placeholder: "Name..." %>
+  <!-- < f.label  :em_contact_phone %> -->
+  <%= f.text_field :em_contact_phone, placeholder: "Phone..." %>
+  <!-- < f.label  :em_contact_relationship %> -->
+  <%= f.text_field :em_contact_relationship, placeholder: "Relationship..." %><br>
+  <%= f.label  :language %>
+  <%= f.text_field :language, class: "form-control" %>
+  <%= f.label  :howd_you_hear? %>
```

Comment:
This question mark needs to be removed so that it matches the actual attribute. You can consider using the syntax above to change the display of the label.

Reply:
I changed the label to `How did you hear?`, but still kept the `?`

Reply:
The intent here is to have the label object *exactly* match the `text_field` object, so you need to object to be *exactly* `howd_you_hear` since that's what being passed to the `text_field`. Where you have more flexibility is in adding a new, separate string parameter, like so:

`<%= f.label  :howd_you_hear, "How did you hear about this program?" %>